packages/common/src/apiutil/functions.ts

```diff
@@ -245,9 +261,8 @@ export function nodesToChunk(nodes: LionWebJsonNode[]): LionWebJsonChunk {
     return {
         serializationFormatVersion: "2023.1",
         languages: collectUsedLanguages(nodes),
-        nodes: nodes,
+        nodes: nodes
     }
 }
 
 export const EMPTY_CHUNK = nodesToChunk([])
```

Comment:
```suggestion
export const EMPTY_CHUNKS: { [K in LionWebVersion]: LionWebJsonChunk } = {
    "2023.1": nodesToChunk([], "2023.1"),
    "2024.1": nodesToChunk([], "2024.1"),
};
```

I would suggest changing EMPTY_CHUNK into a map and update usages from EMPTY_CHUNK to EMPTY_CHUNKS[repositoryData.repository.lionweb_version]

Reply:
Done as suggested